test/Object/rlexe.xml

```diff
@@ -195,6 +195,11 @@
   </test>
   <test>
     <default>
+      <files>getOwnPropertyDescriptors.js</files>
```

Comment:
Make sure to pass "-args summary -endargs" as the flags. That way we can do away with the baseline altogether.